restlicsharpclient/restliclient/transport/DefaultTransportClient.cs

```diff
@@ -59,7 +59,7 @@ private HttpRequestMessage ConstructHttpRequestMessage(HttpRequest httpRequest)
         {
             HttpRequestMessage httpRequestMessage = new HttpRequestMessage()
             {
```

Comment:
This is shorthand object initialization syntax. 

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers